### PR TITLE
previewgif overflow

### LINKED ISSUE #946: Preview gif button overflow.
### Preview GIF overflow.
savebtn

****

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site.

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

### PR DESCRIPTION
**Pull request by Harshithpabbati**
Fixes #946 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask `@publiclab/is-reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

**Comment by Harshithpabbati** [comment]
Now It looks like this 
afterfix

**Comment by Harshithpabbati** [comment]
@publiclab/is-reviewers

**Comment by HarshKhandeparkar** [comment]
Can you show a screenshot of the full page?

**Comment by Harshithpabbati** [comment]
Sorry I can't send it now as I have exam tomorrow.

**Comment by Harshithpabbati** [comment]
over

File: examples/index.html [hunk 1]
```diff
@@ -186,7 +186,7 @@ <h2 style="margin-top:20px">Save</h2>
                 <option>Save sequence string</option>
               </select>
               <button id="saveButton" class="btn btn-primary btn-lg" style="margin-top:20px; margin-bottom:0px;align:center; width:100%;">Save</button>
-              <button class="btn btn-default btn-lg js-view-as-gif" id="gif" style="margin: 0px 35px 0px 35px;width:100%;">Preview GIF</button>
+              <button class="btn btn-default btn-lg js-view-as-gif" id="gif" style="margin-left:0px;margin-top:5px;width:100%;">Preview GIF</button>
```
Comment (HarshKhandeparkar) [inline examples/index.html]: This has a class of 
`.js-view-as-gif`. How about putting the styles in demo.css?

Comment (HarshKhandeparkar) [reply examples/index.html]: You can do the same for save btn as well

**Comment by Harshithpabbati** [comment]
Add the tag 😁

**Comment by HarshKhandeparkar** [comment]
Let someone else review it as well. 2 reviews are necessary.

**Comment by Harshithpabbati** [comment]
p
And even removed the panel down.

**Comment by Harshithpabbati** [comment]
The white panel below the add step one.

**Comment by Harshithpabbati** [comment]
after
This the one after the change.

**Comment by Harshithpabbati** [comment]
Shall I fix the more modules  option???

**Comment by Harshithpabbati** [comment]
@harshkhandeparkar shall I fix the more modules option or is there any other pull request???

**Comment by Harshithpabbati** [comment]
https://github.com/publiclab/image-sequencer/pull/909 more modules has a pull but she is inactive for a week I guess. So I want to make it in a single Pr what do you say???

**Comment by HarshKhandeparkar** [comment]
I have asked the contributor in #909 if they will contribute. If there is no reply, I think assigning the same issue to a different first time contributor will be better. What do you think?

**Comment by Harshithpabbati** [comment]
Yeah ok!!

**Comment by Harshithpabbati** [comment]
@jywarren see this.

**Comment by Harshithpabbati** [comment]
@jywarren merge this.

**Comment by Harshithpabbati** [comment]
@jywarren see this!!!

**Comment by HarshKhandeparkar** [comment]
@Harshithpabbati jywarren must be really busy. Also please note that this branch has conflicts so it cannot be merged automatically.

**Comment by Harshithpabbati** [comment]
Oh fine I just seen the conflicts.

**Comment by jywarren** [comment]
Hi, sorry yes, will be better next week -- had a visit all this week from
some great co-workers and had less reviewing time! Thank you, i tried and
failed to fix this so as soon as it's resolved I'm very happy to merge it,
THANKS!!! :-)

**Comment by Harshithpabbati** [comment]
@jywarren how about increasing the maintainers group??? it will increase the speed.

**Comment by jywarren** [comment]
yes but we would need to be quite strict with the 2 reviews policy, and
perhaps start to be more strict with the tests requirement? Thank you!!

**Comment by Harshithpabbati** [comment]
Yeah what are the things to do to be a maintainer???

File: examples/demo.css [hunk 2]
```diff
@@ -276,4 +276,18 @@ a.name-header{
 .panel-footer{
   height: 45px;
 }
+<<<<<<< HEAD
```
Comment (HarshKhandeparkar) [inline examples/demo.css]: Please fix conflicts

Comment (Harshithpabbati) [reply examples/demo.css]: yeah I am doing them.

**Comment by Harshithpabbati** [comment]
@HarshKhandeparkar done.

File: examples/demo.css [hunk 3]
```diff
@@ -270,10 +268,22 @@ a.name-header{
...
 }
+/*
 .panel-heading{
   height:45px;
 }
 .panel-footer{
   height: 45px;
 }
-*/
+#gif{
```
Comment (HarshKhandeparkar) [inline examples/demo.css]: Why is this commented?

Comment (Harshithpabbati) [reply examples/demo.css]: @jywarren did it in the bootstrap panel.

**Comment by Harshithpabbati** [comment]
@jywarren resolved the conflicts.

File: src/modules/Grid/info.json [hunk 4]
```diff
@@ -0,0 +1,5 @@
+{
+    "name": "Grid Overlay",
```
[the remark this answers is not in the document]
Comment (Harshithpabbati) [reply src/modules/Grid/info.json]: oops thats because of checkout i GUESS

**Comment by Harshithpabbati** [comment]
Done!!!

**Comment by Harshithpabbati** [comment]
My main branch is very clean.

**Comment by Harshithpabbati** [comment]
@jywarren merge this!!

**Comment by Harshithpabbati** [comment]
@jywarren I know you are busy. But please try to merge this. So that the UI looks much good.

**Comment by jywarren** [comment]
Great! Looks good!

> Yeah what are the things to do to be a maintainer???

I think we'd want someone who is a reviewer: https://github.com/publiclab/image-sequencer/issues/656

but then also is able to be responsible for the overall architecture of the system; @tech4GT has been great because he built a lot of the underlying systems. I think perhaps before we expand the maintainers group, we need to develop some shared written principles and goals, and maybe some style guidance for coding, that we can all reference, so it's easier to come to consensus, what do you think? We could pick this discussion up in #656 maybe?

**Comment by Harshithpabbati** [comment]
@jywarren don't merge this

**Comment by Harshithpabbati** [comment]
I need to merge main into this.

**Comment by aashna27** [comment]
I am in the most recent main branch and there is no overflow , although the button is slightly off-positioned!

**Comment by jywarren** [comment]
Yes, i tried fixing it and improved it a little but didn't do nearly as
good a job as Harshith! So, just tell me when it's ready, thank you!!!

**Pull request closed without merging**